actions/restore.py

```diff
@@ -190,11 +191,20 @@ def probe_forked_etcd():
 def reconfigure_client_advertise():
     ''' Reconfigure the backup to use host network addresses for client advertise
         instead of the assumed localhost addressing '''
-    cmd = "/snap/bin/etcd.etcdctl member list"
-    members = check_output(split(cmd))
-    member_id = members.split(b':')[0].decode('utf-8')
 
-    raw_update = "/snap/bin/etcd.etcdctl member update {0} http://{1}:{2}"
+    loop = 0
+    MAX_WAIT = 10
+
+    while loop < MAX_WAIT:
+        try:
+            cmd = "/snap/bin/etcd.etcdctl member list"
+            members = check_output(split(cmd))
+            member_id = members.split(b',')[0].decode('utf-8')
```

Comment:
Why do we now split on comma (`b","`) rather than colon (`b':'`)?  

Has the output of this shell command changed and we need to support two different output formats?

Reply:
I believe this might be an oversight on my side, the `etcdctl` on v3 api uses commas instead of colons in the member list output. Tried utilizing the v2 API with `ETCDCTL_API=2` but couldn't get it working on the `v3` version. Will try a version detection/fallback to old api approach for this.

Reply:
Found the reason why `ETCDCTL_API=2` wasn't working, adding the `--enable-v2` flag was needed on the etcd fork while restoring. Will be going with this fix.